app/Run.hs

```diff
@@ -116,8 +117,8 @@ instance Condense Peer where
     condense (Peer localAddr remoteAddr) = (show localAddr) ++ (show remoteAddr)
 
 
-runNode :: NodeCLIArguments -> LoggingLayer -> IO ()
-runNode nodeCli@NodeCLIArguments{..} loggingLayer = do
+runNode :: NodeCLIArguments -> LoggingLayer -> CardanoConfiguration -> IO ()
+runNode nodeCli@NodeCLIArguments{..} loggingLayer cc = do
     let !tr = (llAppendName loggingLayer) "node" (llBasicTrace loggingLayer)
```

Comment:
I know that this is outside of the scope of this PR, but let me just note that the first pair of brackets is redundant.

Reply:
Done!